### PR TITLE
add new theme

### PR DESCRIPTION
**Pull request by Babel78**
Screenshot of my A11y card:
image

### CODE REVIEW

**Comment by TerabyteTiger** [comment]
Hi! Thanks for submitting a theme and including the A11y card image!

In order to merge your theme, all 3 of the contrast ratios will need to be at least 4.5.

Thanks!

File: src/assets/themes.css [hunk 1]
```diff
@@ -52,3 +52,11 @@ Pull Requests will require passing at least WCAG AA requirements to be accepted
   --link: #82e79a;
   --text: white;
 }
+
+.own-theme{
```
Comment (TerabyteTiger) [inline src/assets/themes.css]: Instead of using `own-theme`, maybe use something like `arduino`, `arduino-theme`, or `arduino-light`?

**Review by TerabyteTiger — CHANGES_REQUESTED** [review]

File: src/assets/themes.css (hunk 1 above)
Comment (Babel78) [reply src/assets/themes.css]: sry

Comment (Babel78) [reply src/assets/themes.css]: image

**Comment by TerabyteTiger** [comment]
Looks awesome! 😎 

Thanks for the contribution! 😄

**Pull request merged**